test/agents/test_conversational_agent.py

```diff
@@ -54,3 +56,78 @@ def test_run():
         agent.run = MagicMock(return_value="Hello")
         assert agent.run("query") == "Hello"
         agent.run.assert_called_once_with("query")
+
+
+@pytest.fixture
+def prompt_node():
+    prompt_node = PromptNode("google/flan-t5-xxl", api_key="fake_key", max_length=256)
```

Comment:
We should just use MockPromptNode as in other Agent tests, otherwise we rely on external APIs.

Reply:
or just use PromptNode() with the default model, which we use in some other tests too